src/libhirte/common/cfg.c

```diff
@@ -137,6 +141,29 @@ int cfg_load_from_file(struct config *config, const char *config_file) {
         return ini_parse(config_file, parsing_handler, config);
 }
 
+char *is_file_name_ending_with_conf(char *file_name, char *conf_file_enfing) {
```

Comment:
is it a type? *conf_file_enfing should it be *conf_file_ending

Reply:
Or maybe just `file_suffix`?

Reply:
Changed  to file_suffix